src/components/CopyInUserDialog.component.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+import BatchModelsMultiSelectModel from "./batch-models-multi-select/BatchModelsMultiSelect.model";
+import BatchModelsMultiSelectComponent from "./batch-models-multi-select/BatchModelsMultiSelect.component";
+import _m from "../utils/lodash-mixins";
+
+function getTitle(getTranslation, users) {
+    const usernames = users && users.map(user => user.userCredentials.username);
+    const info = usernames ? _m.joinString(getTranslation, usernames, 3, ", ") : "...";
+    return getTranslation("copyInUser") + ": " + info;
+}
+
+function getPayload(pairs) {
+    //I need to do this because when the pair maps, there might be more than 1 childUser but only 1 parentUser so
+    //when it iterates the 2nd time for the other child, there is no 2nd parentUser
+    const parent = pairs[0][0]
```

Comment:
If you have to write `pairs[0][0]`, something should be changed. A function should have clean arguments. Here it looks like we need `function getPayload(parentUser, destUsers) { ...`. And then, change the code where getPayload is being called.